proto/cacheclient.proto

```diff
@@ -21,6 +21,7 @@ service Scs {
   rpc Set (_SetRequest) returns (_SetResponse) {}
   rpc SetIfNotExists (_SetIfNotExistsRequest) returns (_SetIfNotExistsResponse) {}
   rpc Delete (_DeleteRequest) returns (_DeleteResponse) {}
+  rpc KeysExists (_KeysExistsRequest) returns (_KeysExistsResponse) {}
```

Comment:
```suggestion
  rpc KeysExist (_KeysExistRequest) returns (_KeysExistResponse) {}
```
for it to read a little more naturally (especially for the poor enunciators among us)?

Reply:
I actually had made this change and updated the doc - did not push the commit yet. Updated just now